Source/Core/Core/Core.cpp

```diff
@@ -374,6 +365,13 @@ static void CpuThread(const std::optional<std::string>& savestate_path, bool del
   MemoryWatcher::Init();
 #endif
 
+  if (savestate_path)
+  {
+    ::State::LoadAs(*savestate_path);
```

Comment:
Do you need the prefix :: ?

Reply:
Yes, otherwise it thinks I mean the enum `Core::State`.